### DISCUSSION TITLE
A way to retrieve slow instances?

### DISCUSSION THREAD

**Discussion by szhorvat** [google/oss-fuzz#11053] [Q&A] [answered]
Is there a way to retrieve inputs that are processed slowly even if it doesn't cause a timeout?

**Comment by DavidKorczynski** [comment, marked as answer]
Not that I'm aware. How about lowering the timeout instead and then trigger a timeout issue? https://github.com/google/oss-fuzz/blob/f724cfcbe1b23980d1ca6c168cdca51eeda19698/projects/harfbuzz/hb-draw-fuzzer.options#L1-L2
